docs/Tutorial_Fast_Approximate_Matrix_Profiles.ipynb

```diff
@@ -257,6 +257,23 @@
     "    plt.show()"
```

Comment:
<p>Minor typo in the inline comment.</p><p> </p><p>change matric to <code>matrix</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/972/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>